cli/tests/common.rs

```diff
@@ -95,15 +97,15 @@ impl TestCli {
     }
 
     pub fn install_extension(&self, path: &Path) -> Assert {
-        self.run(&["extension", "install", "-y", &path.to_string_lossy()])
+        self.cmd().args(&["extension", "install", "-y", &path.to_string_lossy()]).assert()
     }
 
     #[cfg(feature = "extensions")]
     pub fn extension<'a>(&'a self, code: &'a str) -> TestExtensionBuilder<'a> {
         TestExtensionBuilder::new(self, code)
     }
 
-    pub fn run<S: AsRef<str> + AsRef<OsStr>>(&self, args: &[S]) -> Assert {
+    pub fn cmd(&self) -> Command {
```

Comment:
I'm ok with this change but I'd _also_ add a `run` (or more appropriately named) shorthand method to avoid repeating `cmd().args(&[...]).assert()` everywhere.